Sources/SmithyIdentity/AWSCredentialIdentityResolvers/StaticAWSCredentialIdentityResolver.swift

```diff
@@ -22,7 +22,8 @@ public struct StaticAWSCredentialIdentityResolver: AWSCredentialIdentityResolved
         self.crtAWSCredentialIdentityResolver = try AwsCommonRuntimeKit.CredentialsProvider(source: .static(
             accessKey: credentials.accessKey,
             secret: credentials.secret,
-            sessionToken: credentials.sessionToken
-        ), accountId: credentials.accountID)
+            sessionToken: credentials.sessionToken,
+            accountId: credentials.accountID
+        ))
```

Comment:
The interface for accountID on CRT credentials changed slightly since it is now handled on the C side.